lib/sync/index.js

```diff
@@ -45,6 +58,7 @@ function init(dataset_id, options, cb) {
   cb(null, {});
 }
 
+// Stop cloud data sync for the specified dataset_id.
 function stop(dataset_id, cb) {
   syncUtil.doLog(dataset_id, 'info', 'stop');
   
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the useless trailing whitespaces at the end of this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ATrailingWhitespace)